gst/nnstreamer/include/tensor_if.h

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer/NNStreamer Tensor-IF
+ * Copyright (C) 2020 Gichan Jang <gichan2.jang@samsung.com>
+ */
+/**
+ * @file	tensor_if.h
+ * @date	28 Oct 2020
+ * @brief	NNStreamer APIs for tensor_if custom condition
+ * @see		https://github.com/nnstreamer/nnstreamer
+ * @author	Gichan Jang <gichan2.jang@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ * How To for NNdevelopers:
+ *
+ * Define the function in the app.
+ * 1. Define struct, "tensor_if_custom", with the functions defined.
+ * 2. Register the struct with "nnstreamer_if_custom_register" API.
+ * 3. Construct the nnstreamer pipeline and execute it in the app.
```

Comment:
Need more description: a code example on how to define if-condition and how to use it along with tensor_if element.

Reply:
I added the code example. Thanks!